docs/CMakeLists.txt

```diff
@@ -1,5 +1,9 @@
 add_subdirectory(tools)
 
+swift_install_in_component(DIRECTORY diagnostics
+                           DESTINATION "share/doc"
+                           COMPONENT compiler)
```

Comment:
Do we really want to copy files out of the repo's `doc` folder into the install? Most of `doc` is development documentation; it seems strange to mix in end-user documentation too.

Reply:
fair point, maybe there should be a top-level "user-docs" directory. I'll think about it